.github/workflows/core.yml

```diff
@@ -0,0 +1,31 @@
+name: core
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+  build:
+    name: 'linux'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: 'ocaml-multicore/core'
+          ref: 'monorepo-0.14'
+
+      - name: OCaml
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: 'ocaml-variants.4.12.0+domains' 
+          opam-depext: false
+          opam-repositories: |
+            default: https://github.com/ocaml/opam-repository.git
+
+      - run: opam monorepo lock
```

Comment:
If the repository contains `core.opam.locked`, it's better to omit this line, that way you'll get reproducible results. And you can update the dependencies by hand by pushing a new lockfile (obtained by locally running `opam monorepo lock`).

Reply:
Right, anyway I noticed it fails nowadays and cannot recreate a lock file (because of some `ocaml-base-compiler` conflict at some point, I'm not sure I understand it). So it leaves the previous lockfile in place, apparently.
I will remove that line, thanks.